### PR TITLE
add payment tests

### PR DESCRIPTION
**Pull request by AhmedHanafy725**
#391

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 
 tests:
-	pytest tests
+	pytest tests -sv
+
+integrationtests:
+	pytest tests -sv -m "integration"
+
+unittests:
+	pytest tests -sv -m "not integration"
```
Comment (abom) [inline Makefile:10]: I think "unittests" mark would be better instead of integration/not integration?

File: tests/integration/test_payment.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
+import os
+import string
+import time
+
+import pytest
+
+from jumpscale.loader import j
+
+WALLET_NAME = os.environ.get("WALLET_NAME")
```
Comment (abom) [inline tests/integration/test_payment.py:9]: who should set this?

File: tests/integration/test_payment.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    if WALLET_NAME and WALLET_SECRET:
+        wallet = j.clients.stellar.get(WALLET_NAME, network="TEST", secret=WALLET_SECRET)
+        return wallet
+    else:
+        raise ValueError("Please provide add Values to the environment variables WALLET_NAME and WALLET_SECRET")
+
+
+def create_new_wallet():
+    wallet_name = j.data.idgenerator.nfromchoices(10, string.ascii_letters)
+    wallet = j.clients.stellar.new(wallet_name, network="TEST")
```
Comment (abom) [inline tests/integration/test_payment.py:29]: a client is created here and other config objects, do we save them? and do we cleanup later?

Comment (AhmedHanafy725) [reply tests/integration/test_payment.py:29]: No, it's not saved and they will die once the test is finished.

File: tests/integration/test_payment.py (hunk 2 above)
Comment (AhmedHanafy725) [reply tests/integration/test_payment.py:9]: Anyone who wants to run the test, and it will raise an error if it's not set.

**Pull request merged**